bqetl_project.yaml

```diff
@@ -183,6 +183,8 @@ dry_run:
   - sql/moz-fx-data-shared-prod/telemetry/client_probe_counts/view.sql
   - sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_etl_sql_run_check_v1/query.sql
   - sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_scalar_aggregates_v1/query.sql
+  - sql/moz-fx-data-shared-prod/sumo_ga/analytics_314096102/view.sql
```

Comment:
@kwindau ended up skipping these to get CI to pass

Reply:
Amazing, thanks!